src/chrome/content/rules/transfermarkt.de.xml

```diff
@@ -0,0 +1,20 @@
+<!--
+		404:
+			transfermarkt.de
+-->
+<ruleset name="Transfermarkt.de">
+
+	<target host="transfermarkt.de" />
+	<target host="*.transfermarkt.de" />
+		<test url="http://www.transfermarkt.de/" />
+		<test url="http://www.transfermarkt.de/live/counter" />
+		<test url="http://www.transfermarkt.de/navigation/hauptnavi" />
+	
+	<rule from="^http://transfermarkt\.de/"
+			to="https://www.transfermarkt.de/" />
+			<test url="http://transfermarkt.de/" />
```

Comment:
This test url is already covered by the target.

Reply:
https://transfermarkt.de has no content and no redirect to the www-version.  Only http://transfermarkt.de redirects to the www-version. 

What should i do? Remove the target?

Reply:
The target and the rule are fine. Just remove the redundant test url.